index.js

```diff
@@ -5,5 +5,8 @@ export default (element, handler) => {
 	frame.style.cssText = CSS;
 	element.appendChild(frame);
 	frame.contentWindow.onresize = () => { handler(); };
-	return frame;
+	return () => {
+		iframe.contentWindow.onresize = null;
```

Comment:
cool. would be neat to get a definitive answer on this from a browser vendor, though it likely varies. thinking older IE would be the offender.